modules/oe_theme_helper/src/EventSubscriber/IconOptionsEventSubscriber.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\EventSubscriber;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides a list of allowed icons.
+ */
+class IconOptionsEventSubscriber implements EventSubscriberInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      IconOptionsEvent::class => 'getIconOptions',
+    ];
+  }
+
+  /**
+   * Gets the icon options.
+   *
+   * @param \Drupal\oe_paragraphs\Event\IconOptionsEvent $event
+   *   Allowed format event object.
+   */
+  public function getIconOptions(IconOptionsEvent $event): void {
+    $event->setIconOptions([
```

Comment:
But does it make sense to override all icon options in oe_theme? Maybe it is better to first use $event->getIconOptions(), update the received array, and $event->getIconOptions().